anakondo.el

```diff
@@ -197,17 +202,28 @@ and the completion candidates for it at cdr for buffer.")
 
 We try to find the project root by:
 1. Trying to query `clojure-mode' for it.
-2. Trying to query projectile for it.
-3. Defaulting to the `default-directory' of the buffer otherwise.
+2. Trying to query `project' for it.
+3. Trying to query projectile for it.
+4. Defaulting to the `default-directory' of the buffer otherwise.
 
 Anaphoric macro, binds `root' implicitly."
-  `(let* ((root (or (clojure-project-dir)
-                    (projectile-project-root)
+  `(let* ((root (or (and (featurep 'clojure-mode) (clojure-project-dir))
```

Comment:
This seems fine, I just wonder about using `fboundp` versus `featurep`. Any consideration you know off regarding the choice between those two?

Reply:
IIRC featurep should be preferred for such kind of checks, but I don't remember where I've read that. Based on what LSP mode package does, I see that it uses featurep as well in such situations